stl/CMakeLists.txt

```diff
@@ -446,6 +450,8 @@ include_directories(BEFORE
     "${TOOLSET_ROOT_DIR}/crt/src/vcruntime"
 )
 
+add_library(stl_alias_objects OBJECT ${ALIAS_SOURCES})

```

Comment:
After fixing the typo, x86 and x64 are now succeeding, but ARM and ARM64 (which I didn't think to validate locally, oops) are failing with:

```
-- Configuring done
CMake Error: CMake can not determine linker language for target: stl_alias_objects
-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

Reply:
This is because on arm the library has no source files. Before things were added to the same library as a bunch of cpp files, so it wasn't a problem.

Reply:
if cmake doesn't have a language associated with a source file (which it doesn't on arm64 for .asm files) then it just ignores it in the sources target property, but still adds it to the targets in generators like Visual Studio. This is partly historical and partly so you can add headers to your c/c++ targets.